packages/sdk-js/src/mongrule.ts

```diff
@@ -176,22 +184,26 @@ function evalOperatorCondition(
     case "$in":
       if (!Array.isArray(expected)) return false;
       return isIn(actual, expected);
+    case "$inGroup":
+      return isIn(actual, idLists[expected] || []);
+    case "$notInGroup":
+      return !isIn(actual, idLists[expected] || []);
```

Comment:
As far as naming, it's a little weird to use "group" here, but call them "idLists" in the context settings.  I know why it's like this, but now might be a good opportunity to pick better language.  Once we commit to this in the SDK, we can't change it later easily.